tests/jobs/test_scheduler_job.py

```diff
@@ -354,9 +354,10 @@ def test_find_executable_task_instances_pool(self):
         self.assertIn(tis[1].key, res_keys)
         self.assertIn(tis[3].key, res_keys)
 
-    @mock_conf_get('core', 'non_pooled_task_slot_count', 1)
-    def test_find_executable_task_instances_in_non_pool(self):
-        dag_id = 'SchedulerJobTest.test_find_executable_task_instances_in_non_pool'
+    def test_find_executable_task_instances_in_default_pool(self):
+        set_default_pool_slots(1)
+        session = settings.Session()
```

Comment:
```suggestion
```

 ?

Reply:
Let me move this line closer to where it is used.